src/components/Form/Form.js

```diff
@@ -0,0 +1,155 @@
+import React, {Component} from "react";
+import styled from "styled-components";
+import {COLOR_CLAVE_SKIN} from "../../colors";
+import "./error.css"
+
+class Form extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            navn: '',
+            Epost: '',
+            Melding: '',
+            nameValid: false,
+            emailValid: false,
+            messageValid: false,
+            formValid: false,
+            namePlaceholder: 'Navn',
+            emailPlaceholder: 'Epost',
+            messagePlaceholder: 'Melding',
+            nameIsChanged: false,
+            emailIsChanged: false,
+            messageIsChanged: false
+        }
+    }
+
+    handleUserInput = (e) => {
+        const name = e.target.name;
+        const value = e.target.value;
+        this.setState({[name]: value},
+            () => {
+                this.validateField(name, value)
+            });
+    };
+
+    validateField(fieldName, value) {
+        let nameValid = this.state.nameValid;
+        let emailValid = this.state.emailValid;
+        let messageValid = this.state.messageValid;
+
+        switch (fieldName) {
+            case 'Epost':
+                if(value.length > 0 ) {
+                    emailValid = value.match(/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/);
```

Comment:
Bruk heller noe sånn: value.match(/.+@.+/)

https://medium.com/hackernoon/the-100-correct-way-to-validate-email-addresses-7c4818f24643#.5w9tf520m